tests/test_bitbucket_server_webhook.py

```diff
@@ -0,0 +1,95 @@
+from unittest import TestCase
+
+from atlassian.bitbucket import Bitbucket
+
+from .mockup import mockup_server
```

Comment:
Hi @Spacetown 

The mockup_server ImportError is caught in `test_bitbucket_cloud.py`, `test_bitbucket_server.py` and `test_servicedesk.py`.

IMHO, once this error is passed, the object would be None, and the following test must fail.
So I remove the try...except statement. Is it ok?

Is try...except designed for another reason?

Reply:
No, the test will not fail because the is skipped if python < 3.4 where the mockup isn't possible:
```python
@pytest.mark.skipif(sys.version_info < (3, 4), reason="requires python3.4")
```

Reply:
Oh, yes. It will succeed if python < 3.4.
Is this the only reason to introduce try...except?

Reply:
Yes. @gonchik the test don't use this python version anymore. Can this be removed or should it be kept for users developing on python 2.7?